docs/source/how_tos/installation.rst

```diff
@@ -61,13 +69,41 @@ NREL. To query the data you must be on the NREL HPC.
 
 .. todo:: Steps to replicate the database to another system
 
+Apache Spark
+============
+
+- NREL High Performance Computing: :ref:`how-to-start-spark-cluster-eagle`
+- Standalone resources: [TODO: Provide link]
+
 Test your installation
 ======================
 
+If you're running dsgrid at NREL and using the shared registry, you can test your installation 
+with this command:
+
 .. code-block:: console
 
     $ dsgrid -u http://dsgrid-registry.hpc.nrel.gov:8529 -n standard-scenarios registry projects list
 
+You can test your installation similarly if you are using a different registry, just change the 
+ArangoDB URL (-u) and database name (-n) arguments to match your set-up.
+
+Save your configuration
+=======================
+
+Running ``dsgrid config create`` stores key information for working with dsgrid in a config file at 
```

Comment:
This content is good, but perhaps it should be moved to source/reference/cli_fundamentals.rst in the `dsgrid RC file` section and be linked here?

Reply:
I don't feel strongly. I believe I grabbed it from one of the Eagle how-tos.